compiler/src/iree/compiler/Codegen/LLVMGPU/test/matmul_tile.mlir

```diff
@@ -0,0 +1,49 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-lower-executable-target)))" --split-input-file %s
```

Comment:
The test should go in:
https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Codegen/LLVMGPU/test/gpu_set_num_workgroups.mlir